polkadot/node/core/pvf/src/prepare/pool.rs

```diff
@@ -334,22 +334,22 @@ fn handle_mux(
 					handle_concluded_no_rip(from_pool, spawned, worker, idle, result),
 				// Return `Concluded`, but do not kill the worker since the error was on the host
 				// side.
-				Outcome::CreateTmpFileErr { worker: idle, err } => handle_concluded_no_rip(
+				Outcome::CreateTmpFile { worker: idle, err } => handle_concluded_no_rip(
```

Comment:
Not sure if I agree with this change... :thinking: While `PrepareError::CreateTmpFile` is clear, `Outcome::CreateTmpFile` doesn't provide any idea if it's an error condition or not.